layouts/partials/referenced-field-desc-ref-docs.html

```diff
@@ -0,0 +1,28 @@
+{{ $refurl := .url }}
+{{ $refjson := getJSON $refurl }}
+<ul>
+{{ range $key, $value := $refjson.properties }}
+    {{ with $value }}
+        <li><code>{{ $key }}</code> - {{ $value.description}} 
+            {{ with $value.enum }}
+                This must be set to one of the following values:
+                <ul>
+                {{ range $enumval := $value.enum }}
+                    <li>
+                    {{ $enumval }}
+                    </li>
```

Comment:
```suggestion
                    <li> {{ $enumval }} </li>
```
For short ones ... it's easier to read a 1-line ;)

Reply:
Fix other occurrences